src/js/helpers/Util.js

```diff
@@ -119,6 +119,9 @@ var Util = {
   isString: function (obj) {
     return toString.call(obj) === "[object String]";
   },
+  isObject: function (obj) {
+    return toString.call(obj) === "[object Object]";
```

Comment:
What about false positives btw? `Util.isObject(new Number(3))` will return `false`. Add this test:
```
it("handles primitive types", function () {
  expect(Util.isObject(new Number(2))).to.be.true;
});
```
Underscore [does a good old `typeof` check](http://underscorejs.org/docs/underscore.html#section-140), but I've also seen

```
_.isObject = function(obj) {
  return obj === Object(obj);
};
```
Here's some nice info about that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#Description